src/applications/vaos/tests/actions/newAppointment.unit.spec.js

```diff
@@ -217,8 +217,8 @@ describe('VAOS newAppointment actions', () => {
           typeOfCareId: '323',
         },
         pages: {},
-        systemsStatus: FETCH_STATUS.notStarted,
-        systems,
+        parentFacilitiesStatus: FETCH_STATUS.notStarted,
+        parentFacilities: systemsParsed,
```

Comment:
Would it make sense to rename `systemsParsed` to `parentFacilitiesParsed` for clarity/consistency?